code/game/machinery/hologram.dm

```diff
@@ -325,49 +421,27 @@ For the other part of the code, check silicon say.dm. Particularly robot talk.*/
 	hologram.color = color //painted holopad gives coloured holograms
 	set_light(1, 0.1, 2)			//pad lighting
 	icon_state = "[base_icon]1"
-	return 1
+	return TRUE
 
 /obj/machinery/hologram/holopad/proc/clear_holo(mob/living/silicon/ai/user, mob/living/carbon/caller_id)
 	if(user)
 		qdel(masters[user])//Get rid of user's hologram
 		user.holo = null
 		masters -= user //Discard AI from the list of those who use holopad
+
 	if(caller_id)
 		qdel(masters[caller_id])//Get rid of user's hologram
 		masters -= caller_id //Discard the caller from the list of those who use holopad
-	if (!masters.len)//If no users left
+
+	if(!masters.len)//If no users left
```

Comment:
!length(masters)

Reply:
I am being forced to update someone else's code 😔

Reply:
That's called refactoring